lib/matplotlib/backends/backend_agg.py

```diff
@@ -414,6 +415,7 @@ def get_renderer(self, cleared=False):
             self.renderer.clear()
         return self.renderer
 
+    @_api.deprecated("3.8")
```

Comment:
Do we need any of these tostring_ methods?  and even if we do, should they not be private?

Reply:
We could also deprecate tostring_argb (which we don't use either anymore); I don't have a strong opinion here.